NuGet/ChustaSoft.Tools.Authorization/Services/SessionService.cs

```diff
@@ -32,7 +32,7 @@ public class SessionService<TUser> : ServiceBase, ISessionService
         {
             _securitySettings = securitySettings;
             _userService = userService;
-            _tokenHelper = tokenService;            
+            _tokenHelper = tokenService;
```

Comment:
not part of this impl, but I'm realizing that this name is wrong -> tokenHelper param instead of tokenService